tools/openshift/ocp-ipi-aws/prep_for_subm.sh

```diff
@@ -1,82 +1,80 @@
 #!/bin/sh
```

Comment:
You’re adding commands which rely on this being bash, so the shebang should change accordingly:

```suggestion
#!/bin/bash
```